lib/statics/styles.dart

```diff
@@ -72,6 +72,13 @@ class MyStyles {
     fontSize: S4,
     color: Color(MyColors.White),
   );
+  static const whiteMediumUnderlinedTextStyle = TextStyle(
+    fontFamily: "Monument",
```

Comment:
- [ ] Die FontFamiliy würde ich als konstante festlegen, also `static const String kFontFamily = "Monument";` und dann überall hier in der Datei ersetzen.